anaconda_linter/lint/check_completeness.py

```diff
@@ -312,6 +313,21 @@ def check_recipe(self, recipe):
             self.message(section="about", severity=WARNING)
 
 
+class documentation_specifies_language(LintCheck):
+    """Use the generic link not a language specific one
+
+    Please use PACKAGE.readthedocs.io not  PACKAGE.readthedocs.io/en/latest
+
+    """
+
+    def check_recipe(self, recipe):
+        lang_url = re.compile(
+            r"readthedocs.io\/[a-z]{2,3}/latest"
+        )  # assume ISO639-1 or similar language code
+        if recipe.get("about/doc_url", "") and lang_url.search(recipe.get("about/doc_url", "")):
+            self.message(section="about/doc_url")
```

Comment:
```suggestion
            self.message(section="about/doc_url", severity=WARNING)
```

A warning might be a better severity level.